src/providers/MeetingProvider/types.ts

```diff
@@ -53,11 +51,7 @@ export interface MeetingManagerConfig {
   logLevel: LogLevel;
   postLogConfig?: PostLogConfig;
   simulcastEnabled?: boolean;
-  enableWebAudio?: boolean;
+  enableWebAudio?: boolean | undefined;
```

Comment:
Do we even need this `MeetingManagerConfig`? Seem like you can just use `MeetingJoinData`

Reply:
@devalevenkatesh Removed :)

Reply:
@ltrung I think we need `enableWebAudio` param in `MeetingManagerConfig` because this param is provided by the Meeting Provider to the Meeting Manager. I cannot replace `enableWebAudio` with `MeetingSessionConfiguration` properties as meeting session config does not seem to have `enableWebAudio` option. 

Also, `enableWebAudio` is used by the UI in the meeting join screen and we do not have access to the meeting session configuration object at that time.